src/components/modules/FormPage/signUpForm.tsx

```diff
@@ -175,42 +147,39 @@ export default function SignUpForm() {
           What are your Skills? <br />
           (Please enter skills separated with a comma and a space)
         </Label>
-
         <Input id="skills" name="skills" onChange={Formik.handleChange} />
-        {Formik.errors.skills && Formik.touched.skills ? (
-          <ErrorMsg>{Formik.errors.skills}</ErrorMsg>
-        ) : null}
-        <Label>What is your Level of Skill?</Label>
+
+        <Label>What is your level of experience?</Label>
 
         <RadioWrapper>
           <CheckboxLabel>
             <Label>Please Choose One</Label>
           </CheckboxLabel>
           <Row>
-            <Label htmlFor="beginner">Beginner</Label>
+            <Label>Beginner</Label>
             <Radio
-              id="beginner"
+              id="level"
               name="level"
-              value={Level[1]}
+              value={SkillLevel[1]}
               onChange={Formik.handleChange}
             />
           </Row>
 
           <Row>
-            <Label htmlFor="intermediate">Intermediate</Label>
+            <Label>Intermediate</Label>
             <Radio
               name="level"
-              id="intermediate"
-              value={Level[2]}
+              id="level"
+              value={SkillLevel[2]}
```

Comment:
I think we have an issue with the index of every "level" field. they're off by one